### ISSUE TITLE
QuadReadAcross instructions fail validation for compute shader

### ISSUE THREAD

**Issue by xaphier** [microsoft/DirectXShaderCompiler#2302]
According to the documentation of wave intrinsics the "Quad-Wide Shuffle Operations" are available in compute shaders:
[https://github.com/microsoft/DirectXShaderCompiler/wiki/Wave-Intrinsics#quad-wide-shuffle-operations](url)
```
[numthreads(8, 8, 1)] void main()
{
    QuadReadAcrossX(1);
}
```
But validation fails with: 
> opcode 'QuadReadAcross' should only be used in 'Pixel Shader'

**Comment by pow2clk** [comment]
As of today, this is no longer reproducible for me.

**Comment by tex3d** [comment]
Yup, this was fixed by #2361.
